src/targets/gpu/CMakeLists.txt

```diff
@@ -181,18 +316,20 @@ if(MIGRAPHX_ENABLE_MLIR)
 endif()
 
 if(MIGRAPHX_USE_HIPRTC)
+    find_package(hiprtc REQUIRED)
     message(STATUS "MIGraphX is using hipRTC")
     target_compile_definitions(migraphx_gpu PRIVATE -DMIGRAPHX_USE_HIPRTC=1)
 else()
+    find_package(hip REQUIRED)
```

Comment:
This should not be here. We use hip with hiprtc.

Reply:
It is not compiling on Windows when `find_package(hip REQUIRED)` is missing. We will recheck it with a more recent HIP SDK.

Reply:
Noted